manifest.json

```
@@ -1,30 +1,30 @@
 {
-  "serial": 2022042301,
-  "hash": "PGnaDal7sfhKiSTBj6l1oWNH33umpC18YEoIxmhd1oM=",
+  "serial": 2022050501,
+  "hash": "kdFLQNDnNMoIGUGkcsfi+kwL4hvo9kMdvYtBJ6r2650=",
   "signatures": [
     {
       "address": "13YqCei6dP2ibq2DAy81NmU8FYwwzo5HLKXFA3yM3Rvq9WxDfKF",
-      "signature": "mqfXpwwk5USdkI5dNeS4906UFc2DK8b6IYE15gRtK6ASrJBhDvzLm5/Rl6oSa1Y2oSDsQqu0p92cI8E7hl6lCg=="
+      "signature": ""
     },
     {
       "address": "14egjs9cSxt4jnaZTYQE4NemQVXL6qtayqNNpy3VJPSS6AAcaUq",
-      "signature": "3Ggt6p3nMo7NRa52yHBCaiF6QGlqqIniET6KMNgO26cdSn5e0zGD9uMZh4G9l5LvAuVQB27JhVZRn2S8s6UhCA=="
+      "signature": "l6d0YQnGtKTwv9Psc9k/ZGsJklAigkFCs8I06heGgtFu3k6OW9zhQTyI8o/p9KOdV8pBU/RucJotkUPXpxpXAQ=="
     },
     {
       "address": "13RKFFy5qmibyaS6N3Uw18UBDwGhhsXpk6jDn7SMenfDrQ6o6eh",
       "signature": ""
```

```suggestion```
      "signature": "vr4ghxshQw0eOk0BJj/gEZ7rT3wlAZMBWjpA5lP0qCrYWwkA3/oTbRbnJLyf1ysieVw2v7sVSAvwhy5q1vo6CA=="